_test/test_gen_sqw_workflow/gen_sqw_accumulate_sqw_tests_common.m

```diff
@@ -319,10 +319,14 @@ function test_gen_sqw(obj,varargin)
             obj.assertEqualToTolWithSave(w1a,'ignore_str',true,'tol',1.e-7);
         end
         %
-        function DISABLED_test_gen_sqw_sym(obj,varargin)
+        function test_gen_sqw_sym(obj,varargin)
             %-------------------------------------------------------------
+            if obj.save_output
+                return;
+            end
+            skipTest(fprintf('test_gen_sqw_sym_%s is disabled, Ticket #464',obj.test_pref));
```

Comment:
you're skipping the test multiple times: here (always) and below (if bool)

Reply:
this test is currently failing and disabled, so the first sckip. I am starting to work on this ticket. In additional to that, it should be disabled on save as it does not currently save anything (and fails on skip in save mode)

The following skip is there for all tests and this will be there to inform if some tests become disabled on some platform due to some components missing (no mex, no parpool, etc....)